ci_framework/roles/repo_setup/tasks/artifacts.yml

```diff
@@ -19,3 +19,23 @@
     content: |
       {{ _get_hash.stdout | from_json | community.general.json_query('full_hash') }}
     dest: "{{ cifmw_repo_setup_basedir }}/artifacts/repositories/delorean.repo.md5"
+
+- name: Create hash_info file for use in reporting
+  ansible.builtin.copy:
+    content: |
+      DLRNAPI_URL={{ _get_hash.stdout | from_json | community.general.json_query('dlrn_url') }}
+      RELEASE={{ _get_hash.stdout | from_json | community.general.json_query('release') }}
+      FULL_HASH={{ _get_hash.stdout | from_json | community.general.json_query('full_hash') }}
+      {% if cifmw_repo_setup_component is defined %}
+      COMPONENT_NAME={{ _get_hash.stdout | from_json | community.general.json_query('component') }}
+      COMMIT_HASH={{ _get_hash.stdout | from_json | community.general.json_query('commit_hash') }}
+      DISTRO_HASH={{ _get_hash.stdout | from_json | community.general.json_query('distro_hash') }}
+      HASH_ARGS=" --commit-hash $COMMIT_HASH --distro-hash $DISTRO_HASH"
+      {% else %}
+      HASH_ARGS="--agg-hash $FULL_HASH"
+      {% endif %}
+      {% if _get_hash.stdout | from_json | community.general.json_query('extended_hash') is not none %}
+      EXTENDED_HASH={{ _get_hash.stdout | from_json | community.general.json_query('extended_hash') }}
+      HASH_ARGS="$HASH_ARGS --extended_hash $EXTENDED_HASH"
+      {% endif %}
+    dest: "{{ cifmw_repo_setup_basedir }}/artifacts/repositories/hash_info.sh"
```

Comment:
Do we still need this if we are not sourcing it?
Just a reference to debug in jobs?